garage/tf/models/gaussian_mlp_model.py

```diff
@@ -96,7 +114,7 @@ def __init__(self,
 
     def network_output_spec(self):
         """Network output spec."""
-        return ['sample', 'mean', 'std', 'std_param', 'dist']
+        return ['sample', 'mean', 'log_std', 'dist']
```

Comment:
the reason that this is called `std_param` is that the output is not always `log(stddev)`. there are different parameterizations, which required different procedures to recover stddev

Reply:
Ok. I will add that back.